version.php

```diff
@@ -23,7 +23,7 @@
 
 defined('MOODLE_INTERNAL') || die();
 
-$plugin->version   = 2016011300;
+$plugin->version   = 2016042500;
```

Comment:
Yes, looks like it's working! 
But I think we'll still have to do a regression test on a site with the "old" plugin to make sure everything's smooth.